### PR TITLE
Update Retrieving-content-types.md

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
HasChildren

Belongs to https://github.com/umbraco/UmbracoDocs/issues/2759

### CODE REVIEW

File: Reference/Management/Services/ContentTypeService/Retrieving-content-types.md [hunk 1]
```diff
@@ -67,3 +67,18 @@ IEnumerable<IContentType> contentTypes = contentTypeService.GetChildren(Guid.Par
 
 ```
 
+## Check whether a content type has children
+
+In some cases it can be useful to check if a content type has children. The `HasChildren` method can be used to check whether a content type has children.
+
+```C#
```
Comment (sofietoft) [inline Reference/Management/Services/ContentTypeService/Retrieving-content-types.md:74]:
```suggestion
```csharp
```

File: Reference/Management/Services/ContentTypeService/Retrieving-content-types.md [hunk 2]
```diff
@@ -67,3 +67,18 @@ IEnumerable<IContentType> contentTypes = contentTypeService.GetChildren(Guid.Par
...
+// Check if there are children
+var hasChildren = contentTypeService.HasChildren(Guid.Parse("2b54088e-d355-4b9e-aa4b-5aec4b3f87eb"));
+```
+
+Although the use of a GUID is preferable, you can also use it's numeric ID:
+
+```C#
```
Comment (sofietoft) [inline Reference/Management/Services/ContentTypeService/Retrieving-content-types.md:81]:
```suggestion
```csharp
```

**Review by sofietoft — COMMENTED** [review]
Looks good @erikjanwestendorp !
I've made a few suggestions.

Also, would it make sense to mention what type of data the method returns? I'm assuming it's a boolean - true/false.. that might also be implied already 🤔 
What do you think?

**Comment by erikjanwestendorp** [comment]
@sofietoft Thanks for you review. Just updated the PR.

**Review by sofietoft — APPROVED** [review]
Looks great! I'll get it merged it 💪

**Pull request merged**